test/test_find_uvar.py

```diff
@@ -114,15 +120,16 @@ def test_two_triangles_without_faces():
     grid = two_triangles_with_depths()
     grid.faces = None
 
-    grid.save_as_netcdf('2_triangles_without_faces.nc')
-
-    # Read it back in and check it out.
-    ug = UGrid.from_ncfile('2_triangles_without_faces.nc', load_data=True)
+    fname = '2_triangles_without_faces.nc'
+    with chdir(test_files):
+        grid.save_as_netcdf(fname)
+        ug = UGrid.from_ncfile(fname, load_data=True)
+        os.remove(fname)
```

Comment:
Why not? in the CI it really doesn't matter if you clean up after yourself or not, as teh environment will get thrown away anyway. (same for conda build, etc...)

When developing, I find it handy for generated files to stick around so I can examine them. And it isn't really a problem to have them stick around, unless someone does the unforgivable `git add *`. Though you do get bunch of files showing up  as untracked by git, which can be annoying.

But this method makes it easy to comment out the `os.remove` when debugging, so not a bad way to go, either. So let's stick with this for now.